spring-boot/admin/src/test/resources/bootstrap.properties

```
@@ -0,0 +1,8 @@
+spring.application.name=ygg-admin
+spring.cloud.config.uri=https://config.coatrack.eu
```

To make it easier understandable for the reader, maybe we should make clear that these are dummy values for the test. E.g.

```suggestion
spring.cloud.config.uri=https://test-dummy-config-url.coatrack.eu
```

---

Done, and it worked well